### DISCUSSION TITLE
New CIS Benchmark version 3.1.1 for RHEL7 and 1.0.1 for RHEL8

### DISCUSSION THREAD

**Discussion by durkacivan33** [ComplianceAsCode/content#7062] [Q&A] [answered]
Hi,
 I would like to ask if there is a plan to create a new content in openscap for the new CIS benchmark release. Currently there is f.e. for RHEL7 version 3.1.1, but with openscap I can run scan/remediations only with version 2.2.0.

Thanks in advance for answer

**Comment by marcusburghardt** [comment, marked as answer]
Hi @durkacivan33 , currently (at the moment of this comment), the CIS profile for RHEL7 is already in alignment to the last CIS version:
https://github.com/ComplianceAsCode/content/blob/master/controls/cis_rhel7.yml#L4

Regarding RHEL8, it is currently one version behind:
https://github.com/ComplianceAsCode/content/blob/master/controls/cis_rhel8.yml#L5

But should be alignment to the v2.0.0 within the next 3 months.
